website/source/docs/secrets/gcp/index.html.md

```diff
@@ -108,21 +127,24 @@ was configured, you can generate OAuth2 tokens or service account keys.
 To generate OAuth2 tokens, read from `gcp/token/...`. The roleset must have been
 created as type `access_token`:
 
-```text
-$ vault read gcp/token/my-token-roleset
-
-Key                Value
----                -----
-lease_id           gcp/token/my-token-roleset/a8e792ec-9080...
-lease_duration     60m
-lease_renewable    false
-token              ya29.c.Elq3BWCRTcuzAyjYDlJZ2tA1zx_7O5AOL...
-```
+    ```text
+    $ vault read gcp/token/my-token-roleset
+    
+    Key                Value
+    ---                -----
+    expires_at_seconds    1537402548
+    token                 ya29.c.ElodBmNPwHUNY5gcBpnXcE4ywG4w1kmyfaketoken...
+    token_ttl             3599
+    ```
 
-This endpoint generates a non-renewable OAuth2 access token with the specified TTL. The `token` is a GCP OAuth2 access token that can be used as a Bearer token as part of an API call to Google Cloud.
+This endpoint generates a non-renewable, non-revocable static OAuth2 access token
+with a lifetime of one hour, where `token_ttl` is given in seconds and the 
+`expires_at_seconds` is the expiry time for the token, given as a Unix timestamp. 
+The `token` value then can be used as a HTTP Authorization Bearer token in requests 
+to GCP APIs: 
 
 ```sh
-$ curl -H "Authorization: Bearer ya29.c.Elq3BWCRTcuzAyjYDlJZ2tA1zx_7O5AOL..."
+$ curl -H "Authorization: Bearer ya29.c.ElodBmNPwHUNY5gcBpnXcE4ywG4w1kmyfaketoken..."
```

Comment:
myfaketoken?

Reply:
i changed the value but added some fake stuff there - can remove